grakn-engine/src/test/java/ai/grakn/engine/EngineTestHelper.java

```diff
@@ -81,4 +108,60 @@ public static synchronized void noEngine() {
             server = null;
         }
     }
+    
+    public static synchronized GraknEngineServer graknEngineServer(GraknEngineConfig config) {
+        RedisWrapper redisWrapper = redisWrapper(config);
+        return EngineTestHelper.graknEngineServer(config,redisWrapper);
+    }
+
+    private static RedisWrapper redisWrapper(GraknEngineConfig config) {
```

Comment:
To eliminate all of this duplicated code might it not be best to expose the relevant methods in `GraknCreator` or provide some common factory methods? 

I imagine this being problematic to maintain long term.

Reply:
I put the reason of this duplication in the first comment, let me know you if it convinces you .. ;)
Otherwise I will just create private method and let EngineTestHelper.java extend GraknCreator

> I duplicated a bit the creation code also in 'EngineTestHelper' because integration tests need fresh instances (not the already-used ones, learnt in the hard-way after so many test failures ..).
I could have put those methods in 'GraknCreator', but then it was not clear anymore for a user of 'GraknCreator' which one to use to get an instance of a specific class. This decision forced me to add a new test dependency between 'grakn-engine' module and 'grakn-test', that has only 'test' as a scope. Final thought: I think that duplication is better than risk of misuse!

Reply:
I read that and because of how we work I am not entirely. Each component is very much "owned" by someone so the risk of misuse is less (in my opinion) than the risk of forgetting to maintain the duplicated code. Especially because of the rate of flux on how we change these components.

Reply:
ok got it! I will change!
Thanks for the overview!